scripts/maint/post-merge.git-hook

```diff
@@ -13,12 +13,20 @@ check_for_diffs() {
                         echo "ATTENTION: $1 hook has changed:"
                         echo "==============================="
                         diff "$installed" "$latest"
-                        exit 1
                fi
         fi
 }
 
+check_for_script_update() {
+        fullpath="$git_toplevel/scripts/maint/$1"
+
+        git diff ORIG_HEAD HEAD --exit-code -- "$fullpath"
+}
+
 check_for_diffs "pre-push"
 check_for_diffs "pre-commit"
 check_for_diffs "post-merge"
 
+check_for_script_update "git-merge-forward.sh"
```

Comment:
Can we use the strategy of "check_for_script_update" for all of the files?

Reply:
Maybe we should move all of these files into a scripts/git directory, and apply this hook to every file in the directory.  That way, it won't need to list all of the hooks and scripts that we use

Reply:
Moving the script files to new directory in 3e895fbb504edea174d21b6e8c8b4be02cd636c7 and iterating through them in 7f87393c6c8bc9eee81ab2ef4643112cd012c5d9.